apps/com.microsoft.Skype/eos-skype.appdata.xml

```diff
@@ -4,4 +4,7 @@
   <categories>
     <category>Blacklisted</category>
   </categories>
+  <metadata>
+    <value key="Endless::ReplacedByDesktopFile">com.microsoft.Skype.desktop</value>
```

Comment:
This does not seem to match EndlessOS::replaced-by-desktop-file used in https://github.com/endlessm/gnome-software/pull/244